### ISSUE TITLE
LocalDateTime regression

### ISSUE THREAD

**Issue by park-infox** [JetBrains/Exposed#1028]
Only the millisecond is stored in the `TIMESTAMP` OracleDatabase  column
Same as #1008 but not fixed in my environment

Example

Before (in 0.26.1)
**2020-08-30 12:35:03.172357**

After (0.26.2 or later)
**2020-08-30 12:35:03.172**

--

**Environment**

OpenJDK 64-Bit Server VM 18.9 (build 11.0.2+9, mixed mode)
Oracle Database 11g

**Comment by ps-feng** [comment]
Sorry I didn't follow up on the original issue but it's still happening with MySQL as well.

**Comment by andreysoldatenkow** [comment]
Any update on this?
Running into the same issue updating to 0.31.1 from version 0.24.1
OS: Ubuntu 20.10
Postgres version: 9.6.21
Java version: 11
Kotlin version: 1.5.0

**Comment by Tapac** [comment]
Hello @park-infox , @ps-feng , @andreysoldatenkow , can you specify what module do you use `exposed-jodatime` or `exposed-java-time`?

**Comment by Tapac** [comment]
Looks like I found and fix the problem. Will be released soon.

**Comment by park-infox** [comment]
@Tapac Tapac
I'm use `exposed-java-time`

**Comment by ChristianSch** [comment]
@Tapac I'm still experiencing this. We're using 0.32.1 and I thought that this problem should be fixed in said version. @SchweinchenFuntik and you seemed to have made commits somewhere else. What's the status of the problem?

```
expected to contain exactly in any order:<[Role(id=1, createdAt=2021-11-04T15:49:31.245766+01:00[Europe/Berlin], updatedAt=2021-11-04T15:49:31.245776+01:00[Europe/Berlin], name=TEST1), Role(id=2, createdAt=2021-11-04T15:49:31.255858+01:00[Europe/Berlin], updatedAt=2021-11-04T15:49:31.255867+01:00[Europe/Berlin], name=TEST2)]> but was:<[Role(id=1, createdAt=2021-11-04T15:49:31.245+01:00[Europe/Berlin], updatedAt=2021-11-04T15:49:31.245+01:00[Europe/Berlin], name=TEST1), Role(id=2, createdAt=2021-11-04T15:49:31.255+01:00[Europe/Berlin], updatedAt=2021-11-04T15:49:31.255+01:00[Europe/Berlin], name=TEST2)]>

```

**Comment by Tapac** [comment]
@ChristianSch , the fix should be available since [#0.33.1](https://github.com/JetBrains/Exposed/blob/master/docs/ChangeLog.md#0331)
